contracts/Vault.vy

```diff
@@ -1273,7 +1290,7 @@ def report(_return: uint256) -> uint256:
 
     log StrategyReported(
         msg.sender,
-        _return,
+        _gain,
```

Comment:
do we want to track reported loss value in event as well?

Reply:
Yes, I'll probably change up the events in a later PR too